projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.html

```diff
@@ -35,7 +35,7 @@
       </mat-expansion-panel-header>
       <data-layer-options [layer]="layer" (layerChange)="onLayerChange($event)"></data-layer-options>
       <mat-action-row>
-        <button mat-button color="primary" (click)="onLayerRemove(layer)" title="Remove this layer"><mat-icon fontIcon="delete"></mat-icon> REMOVE LAYER</button>
+        <button mat-button color="primary" (click)="onLayerRemove(layer)" title="Remove this layer" [hidden]="showRemoveButton"><mat-icon fontIcon="delete"></mat-icon> REMOVE LAYER</button>
```

Comment:
It looks like `showRemoveButton` is doing the opposite of what it says

Reply:
okay @semicolin